README.md

```diff
@@ -26,6 +28,9 @@ handler which traces server requests.
 ```java
 Router router = Router.router(vertx);
 
+Tracer tracer = Configuration.fromEnv().getTracer();
```

Comment:
This is Jaeger specific we cannot have it here.

Reply:
Sorry @pavolloffay, wrong place!